models/avatar_test.go

```diff
@@ -41,7 +41,7 @@ func TestHashEmail(t *testing.T) {
 
 func TestSizedAvatarLink(t *testing.T) {
 	disableGravatar()
-	assert.Equal(t, "/suburl/assets/img/avatar_default.png",
+	assert.Equal(t, "/assets/img/avatar_default.png",
```

Comment:
For this one it would be useful to add a suburl to appurl so that we can assert that the suburl is present in the results.

A common and repeated bug is people failing to add AppSubUrl to their urlpaths.